src/ledger.ts

```diff
@@ -180,16 +163,14 @@ export class LedgerInteraction extends DirectKeystoreInteraction {
    *
    * See the [Ledger API]{@link https://github.com/LedgerHQ/ledgerjs} for general information or a [specific transport API]{@link https://github.com/LedgerHQ/ledgerjs/tree/master/packages/hw-transport-u2f} for examples of API calls.
    *
-   * @param {function} callback -- asynchronous function accepting a single parameter `transport`
-   * @returns {Promise} does the work of setting up a transport connection
    * @example
    * async run() {
    *   return await this.withTransport(async (transport) => {
    *     return transport.doSomething(); // Not a real Ledger transport API call
    *   });
    * }
    */
-  async withTransport(callback) {
+  async withTransport(callback: (transport: any) => any) {
```

Comment:
I wonder if we can get the transport type from the ledger library.

Reply:
I tried pulling it out of AppClient, but it doesn't seem to be made available.